src/barkha/Assignment1.java

```diff
@@ -0,0 +1,96 @@
+/*1) Open the form in browser "file:///C:/Users/DELL%201801/Downloads/Microsoft.SkypeApp_kzf8qxf38zg5c!App/All/SeleniumAssignment_1%20(3).html" and verify Title is �TECHNOCREDITS�. 
+
+2) Fill all details in the form, for Gender radio button if �Female� radio button is by default selected then select �Male�   radio button and vice versa.
+
+3) Reset the form, for that click on �Reset form in same tab� button.
+
+4) Verify Reset works properly on your text field, radio button and checkbox. (For each type, min one validation is required.)
+
+5) Click on the �Go and Practice for it� Button.
+
+6) Website redirect to �automationbykrishna.com�.
+
+7) Verify title and URL of redirected page website.
+    Title should be �Login Signup Demo� and the URL should be �http://automationbykrishna.com/�.
+*/
+package barkha;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Assignment1 {
+	
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", "./resources/windows/chromedriver.exe");
+		WebDriver driver=new ChromeDriver();
+		driver.get("file:///C:/Users/DELL%201801/Downloads/Microsoft.SkypeApp_kzf8qxf38zg5c!App/All/SeleniumAssignment_1%20(3).html");
+		
+		//verify correct title
+		if (driver.getTitle().equals("TECHNOCREDITS"))
+			System.out.println("You are on a correct Page having title TECHNOCREDITS.");
+		else
+			System.out.println("You are on a incorrect page.");
+		
+		//filling all details in the form
+		WebElement firstName=driver.findElement(By.id("first name"));
+		firstName.sendKeys("Barkha");
+		
+		WebElement lastName=driver.findElement(By.id("last name"));
+		lastName.sendKeys("Patle");
+		
+		WebElement email=driver.findElement(By.id("E-mail"));
+		email.sendKeys("barkhapatle01@gmail.com");
+		
+		WebElement company=driver.findElement(By.id("Company Name"));
+		company.sendKeys("Syntel");
+		
+		//if female radio button is selected then click on male and vice versa
+		if (driver.findElement(By.id("femaleG"))!=null) {
```

Comment:
Here you should use isSelected() method.

Reply:
Code changes done as suggested